### ISSUE TITLE
About Training!

### ISSUE THREAD

**Issue by chengfeng0113** [abrosua/piv_liteflownet-pytorch#2]
There is a bug in the code, and the training does not converge. There is a problem with the loss function. It is recommended that you upload the code after the training is OK, otherwise it will be a waste of time.
